functions/go/search-replace/searchreplace/searchreplacecases_test.go

```diff
@@ -727,4 +727,51 @@ spec:
  `,
 		errMsg: `only one of ["by-value", "by-value-regex"] can be provided`,
 	},
+	{
+		name: "error when none of the search matchers are provided",
+		config: `
```

Comment:
don't use `config` as a name anywhere. Everything's a config, so it's meaningless. I think you want `functionConfig` here.

Reply:
All the other functions tests use the same name for `config`. Since these are internal tests and not exposed to users, we can unblock this PR and have a different issue to change to search for such references and change everywhere.

https://github.com/GoogleContainerTools/kpt/issues/1883